trollmoves/dispatcher.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2012-2019
+#
+# Author(s):
+#
+#   Martin Raspaud <martin.raspaud@smhi.se>
+#   Panu Lahtinen <panu.lahtinen@fmi.fi>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Dispatcher dispatches(!) files to some destination.
+
+It listens to posttroll messages to find out what to dispatch.
+
+
+Format of the configuration file
+================================
+
+.. highlight:: yaml
+
+Example config::
+
+    client1:
```

Comment:
I believe this is just a generic name for the configuration section? Could it be renamed to something like `target_host_1_config` to distinguish from `client` side of `trollmoves`.

Reply:
Good point. Something like this ?
```suggestion
    target1:
```

Reply:
Yes, that's more clear.